src/openforms/forms/migrations/0098_fix_forms_conditionals.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.11 on 2024-04-16 12:59
+
+from django.db import migrations
+
+from openforms.formio.migration_converters import (
+    convert_simple_conditionals,
+)
+
+
+def fix_forms_simple_conditionals(apps, schema_editor):
+    FormDefinition = apps.get_model("forms", "FormDefinition")
+
+    all_definitions = FormDefinition.objects.all()
+    if not len(all_definitions):
+        return
+
+    definitions_to_update = []
+    for definition in all_definitions:
```

Comment:
(for future, this is fine for this PR)

1. try to favour `.iterator()` over `.all()` in data migrations when operating on potentially large datasets. it prevents the entire queryset from being loaded in memory and thus won't crash your application with OOM errors
2. the `len(all_definitions)` check isn't needed - you can replace it with a `queryset.exists()` for an optimized query if you absolute must prevent work being done, but in this case, I would just loop over the (empty) qs and only do the `bulk_update` if `definitions_to_update` is not empty. I wouldn't be surprised if django itself already skips any work to be done if an empty list is passed.